tutorials/using-xgboost/2.distributed-cpu.ipynb

```diff
@@ -0,0 +1,156 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Distributed XGBoost (CPU)\n",
+    "\n",
+    "Scaling out on AmlCompute is simple! The code from the previous notebook has been modified and adapted in [src/run.py](src/run.py). In particular, changes include:\n",
+    "\n",
+    "- use ``dask_mpi`` to initialize Dask on MPI\n",
+    "- use ``argparse`` to allow for command line argument inputs\n",
+    "- use ``mlflow`` logging \n",
+    "\n",
+    "The [environment.yml](environment.yml) contains the conda environment specification."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Get Workspace"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from azureml.core import Workspace\n",
+    "\n",
+    "ws = Workspace.from_config()\n",
+    "ws"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Distributed Remotely\n",
+    "\n",
+    "Simply use ``MpiConfiguration`` with the desired node count."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "nodes = 8 + 2\n",
+    "cpus_per_node = 4\n",
+    "\n",
+    "print(f\"Nodes: {nodes}\\nCPUs/node: {cpus_per_node}\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "arguments = {\n",
```

Comment:
why are you using a dict here?

Reply:
more logical to work with arguments in a dict than in a flat list

Reply:
switched to list